### DISCUSSION TITLE
packages not recognized inside a python file. not able to run python files

### DISCUSSION THREAD

**Discussion by reachpalaniv** [microsoft/vscode-discussions#980] [Extension Development QnA] [answered]
I am running a venv using python 3.11.7 version inside vs code. I installed numpy, pandas and torch but i am not able to run any python file since the packages are not getting recognized. I tried disabling the python extensions to see if that is causing an issue. I need help to resolve this issue since I am stuck for the past 3 days. please help. I am on a brand new Mac book pro running latest Sonoma 14.2

**Comment by gjsjohnmurray** [comment, marked as answer]
Please review the [READ FIRST post](https://github.com/microsoft/vscode-discussions/discussions/5) that is pinned to the top of this discussion forum. It explains the current focus of the forum, which your post doesn't fit with. It also signposts some other resources.
